omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,103 @@
+Complete workflow for hosting data for a publication
+====================================================
+
+The following describes the steps of a complete workflow for using OMERO to
+host public data associated with a publication. It is illustrated using an
+example publication from our lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_.
```

Comment:
After following that link and browsing around for a while, I still can't find the public OMERO data, which is probably what users would be most interested in seeing? Can we point to the data here, or have both links to the data and the paper?

Reply:
There are extensive links to the data lower down the page, I just stuck the paper link here because it didn't seem worth leaving it out.

Reply:
Yes, I found them later, but only after I gave up trying to find them from this link.

Reply:
It is there under Data accessibility paragraph, @will-moore . Admittedly, I never found it myself when I saw the page.